t/02-rakudo/15-gh_1202.t

```diff
@@ -0,0 +1,26 @@
+use v6;
+use Test;
+
+# GH#1202
+
+plan 2;
+
+my Bool $complete = False;
+
+await Promise.anyof(
+    start {
+        my $out = "";
+        # This was a bug with unpredictable output. Hopefully 50 repetiotions is enough to make sure everything works.
+        for 1..50 -> $rep {
+            my $proc = run $*EXECUTABLE, '-MTest', '-e', q<await (^5).map({start { print qqx{echo $_} } })>, :out;
+            $out ~= $proc.out.slurp;
```

Comment:
Change to `$proc.out.slurp(:close)` or add a`$proc.out.close`

Reply:
Ok. Thanks!